packages/web/src/components/basic/ToggleButton.js

```diff
@@ -173,12 +173,12 @@ class ToggleButton extends Component {
 					&& <Title className={getClassName(this.props.innerClass, 'title') || null}>{this.props.title}</Title>
 				}
 				{this.props.data.map((item) => {
-					const isSelected = this.state.currentValue.some(value => value.label === item.label);
+					const isSelected = this.state.currentValue.some(value => value.value === item.value);
 					return (
 						<Button
 							className={`${getClassName(this.props.innerClass, 'button')} ${isSelected ? 'active' : ''}`}
 							onClick={() => this.handleToggle(item)}
-							key={item.label}
+							key={item.value}
```

Comment:
@metagrover @divyanshu013 
If we allow duplicate `value`, what should this be changed to?

Reply:
Ideally, you shouldn't have duplicate `value` keys in the `data` prop. So, we can ignore this until a use-case comes up.